### PR TITLE
Upgrade deps

### PR DESCRIPTION
**Pull request by mischnic**
~Blocked by https://github.com/parcel-bundler/lightningcss/issues/325~

The big breaking changes were:
- Newer (more strict) version of lightningcss
- Tailwind 3

### CODE REVIEW

File: src/assets/styles/doc-styles.scss [hunk 1]
```diff
@@ -739,13 +739,15 @@ body {
...
     }
 
+    /*
     @page :left {
       margin: 15mm 20mm 15mm 10mm;
     }
 
     @page :right {
       margin: 15mm 10mm 15mm 20mm;
     }
```
Comment (mischnic) [inline src/assets/styles/doc-styles.scss:749]: Not sure what that's even supposed to mean, compiled to css:
```css
@media print {
  @page :left {
    .main-content {
      margin: 15mm 20mm 15mm 10mm;
    }
  }
}
```

File: .parcelrc [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
   "extends": "@parcel/config-default",
   "resolvers": ["...", "parcel-resolver-website"],
-  "namers": ["parcel-namer-website", "..."],
   "transformers": {
-    "*.css": ["@parcel/transformer-postcss", "@parcel/transformer-css-experimental"]
+    "*.{js,mjs,jsm,jsx,es6,cjs,ts,tsx}": [
+      "@parcel/transformer-js",
+      "@parcel/transformer-react-refresh-wrap"
+    ]
```
Comment (devongovett) [inline .parcelrc:8]: Why did we need to override the js pipeline?

File: src/assets/styles/doc-styles.scss (hunk 1 above)
Comment (devongovett) [reply src/assets/styles/doc-styles.scss:749]: Theoretically if you printed the website, and made a two page spread, the left page would have different margin from the right page. But idk where this came from. What was broken with it?

File: .parcelrc (hunk 2 above)
Comment (mischnic) [reply .parcelrc:8]: [generate-api-docs](https://github.com/parcel-bundler/website/tree/v2/generate-api-docs) is written using flow, so the .flowconfig caused every file to be transpiled with Babel.

**Review by devongovett — APPROVED** [review]

**Pull request merged**